src/nccl_ofi_rdma.c

```diff
@@ -130,39 +130,24 @@ static pthread_mutex_t topo_file_lock = PTHREAD_MUTEX_INITIALIZER;
 /*
  * @brief	Return input tag indicating data path message
  */
-#define GET_DATA_MSG_TAG(tag) (((tag) & ~TAG_TYPE_TAG_MASK) | DATA_MSG_TYPE_MASK)
+#define GET_DATA_MSG_TAG(tag) ((tag << NUM_TAG_TYPE_BITS) | CONN_MSG_TYPE_MASK)
```

Comment:
The argument here should be `comm_id` now, rather than `tag`, to reflect what you are passing into it. Same in the other macros.